app/components/draft/draft.test.tsx

```diff
@@ -0,0 +1,190 @@
+            channel: {type: General.OPEN_CHANNEL, displayName: 'Direct Message Channel'} as ChannelModel,
+            location: 'channel',
+            draft: {
+                updateAt: 1633024800000,
+                message: 'Hello, World!',
+                channelId: 'channel_id',
+                rootId: '',
+                files: [],
+                metadata: {},
+            } as unknown as DraftModel,
+            layoutWidth: 100,
+            isPostPriorityEnabled: false,
+        };
+        const wrapper = renderWithEverything(
+            <Draft
+                channel={props.channel}
+                location={props.location}
+                draft={props.draft}
+                layoutWidth={props.layoutWidth}
+                isPostPriorityEnabled={props.isPostPriorityEnabled}
+            />
+            , {database},
+        );
+
+        const {getByText} = wrapper;
+
+        expect(FormattedText).toHaveBeenCalledWith(
+            expect.objectContaining({
+                id: 'channel_info.draft_in_channel',
+                defaultMessage: 'In:',
+            }),
+            expect.anything(),
+        );
+        expect(CompassIcon).toHaveBeenCalledWith(
+            expect.objectContaining({
+                name: 'globe',
+            }),
+            expect.anything(),
+        );
+        expect(getByText('Hello, World!')).toBeTruthy();
+        expect(wrapper.toJSON()).toMatchSnapshot();
+    });
+
+    it('should match the file count', () => {
+        const props = {
+            channel: {type: General.OPEN_CHANNEL, displayName: 'Direct Message Channel'} as ChannelModel,
+            location: 'channel',
+            draft: {
+                updateAt: 1633024800000,
+                message: 'Hello, World!',
+                channelId: 'channel_id',
+                rootId: '',
+                files: [{
+                    has_preview_image: false,
+                    height: 0,
+                    name: 'file1.txt',
+                    extension: 'txt',
+                    size: 64,
+                }, {
+                    has_preview_image: false,
+                    height: 0,
+                    name: 'file2.pdf',
+                    extension: 'txt',
+                    size: 64,
+                }],
+                metadata: {},
+            } as unknown as DraftModel,
+            layoutWidth: 100,
+            isPostPriorityEnabled: false,
+        };
+        const wrapper = renderWithEverything(
+            <Draft
+                channel={props.channel}
+                location={props.location}
+                draft={props.draft}
+                layoutWidth={props.layoutWidth}
+                isPostPriorityEnabled={props.isPostPriorityEnabled}
+            />
+            , {database},
+        );
+        const {getAllByTestId} = wrapper;
+        expect(getAllByTestId('file_attachment')).toHaveLength(2);
+        expect(wrapper.toJSON()).toMatchSnapshot();
```

Comment:
Same as above:
```suggestion
        const {getAllByTestId, toJSON} = renderWithEverything(
            <Draft
                channel={props.channel}
                location={props.location}
                draft={props.draft}
                layoutWidth={props.layoutWidth}
                isPostPriorityEnabled={props.isPostPriorityEnabled}
            />
            , {database},
        );
        expect(getAllByTestId('file_attachment')).toHaveLength(2);
        expect(toJSON()).toMatchSnapshot();
```

Reply:
Same question throughout, how would the snapshot help understand that there are two files?